contributions/catalog/webapps/templates/contribute/home.html

```diff
@@ -246,13 +246,8 @@
 <div class="navbar" id="myTopnav">
     <a href="{{ url_for('contribute.home') }}">Contribution Catalog</a>
     <a href="{{ url_for('contribute.create') }}">Submit</a>
-    <script>console.log(state)</script>
-    <a href="{{ url_for('handleLogin') }}" onsubmit="">login</a>
-    <!--    <input id="searchbar" type="text" placeholder="Search by ID...">-->
-    <!--    <button id="myBtn" style="display:none">Button</button>-->
-    <!--    <div class="search-container">-->
+    <a >hello, @{{ user }}</a>
```

Comment:
Same here too. Also, I think we need to add this to base.html so that if name is available in the session, it is displayed in all pages after login.

Reply:
@wenjzhu, do you plan to move the greetings to base.html in this PR? If so, please do that. If not, please update the CHANGELOG and I think we can merge this PR. Thanks.

Reply:
Updated, I will create a new pr for refactoring greeting and nav bar into base.html.